mail_layout_preview/wizard/email_template_preview.xml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
-  <record model="ir.ui.view" id="email_template_preview_form">
```

Comment:
This formatting change should be in the commit [[IMP] mail_layout_preview: pre-commit auto fixes](https://github.com/OCA/mail/pull/7/commits/a611afe200be7ecc0db0158bfb6c384e0d04d4be)
IMO